libs/application/templates/estate/src/forms/ResidencePermit/overviewSection.ts

```diff
@@ -154,17 +159,23 @@ export const overview = buildSection({
           marginBottom: 'gutter',
           space: 'gutter',
         }),
-        buildKeyValueField({
-          label: m.bankAccount,
-          value: ({ answers }) =>
-            (answers.bankAccounts as any)[0].accountNumber,
-          width: 'half',
-        }),
-        buildKeyValueField({
-          label: m.bankAccountBalance,
-          value: ({ answers }) => (answers.bankAccounts as any)[0].balance,
-          width: 'half',
-        }),
+        buildCustomField(
+          {
+            title: '',
+            id: 'bankAccountsCards',
+            component: 'Cards',
+            doesNotRequireAnswer: true,
+          },
+          {
+            cards: ({ answers }: Application) =>
```

Comment:
The answers are validated by the dataschema.
The dataschema can be used to infer types.

Maybe we could use it to type the answers rather than using any

```typescript
import { estateSchema } from '@island.is/application/templates/estate'
import { infer as zinfer } from 'zod'
type EstateSchema = zinfer<typeof estateSchema>
type EstateAnswers = EstateSchema['estate']

// and then later
cards: ({ answers }: EstateAnswers) =>
  (answers?.bankAccounts ?? []).map((account) => .....
```